app/assets/stylesheets/application.scss

```diff
@@ -15,3 +15,31 @@
  *= require foundation_and_overrides
 
  */
+
+ .profile-photo img {
+  width: 3em;
+  max-height: 3em;
+  float: right;
+  margin-top: 2px;
+  margin-right: 0px;
+  background: inherit;
+}
+
+.profile-photo li {
+  background: #000000;
+}
+
+.user-pic img {
+ width: 3em;
+ max-height: 3em;
+ margin-top: 2px;
+ margin-right: 0px;
```

Comment:
Line should be indented 2 spaces, but was indented 1 spaces<br>`0px` should be written without units as `0`